chaincfg/v1.5.0 specifies too low a version of edwards

```
$ GO111MODULE=on go get github.com/decred/dcrd/chaincfg/chainec@v1.5.0    
go: finding github.com/decred/dcrd/chaincfg/chainec v1.5.0
# github.com/decred/dcrd/chaincfg/chainec
go/pkg/mod/github.com/decred/dcrd/chaincfg@v1.5.0/chainec/edwards.go:278:30: not enough arguments in call to edwards.GenerateKey
        have (io.Reader)
        want (*edwards.TwistedEdwardsCurve, io.Reader)
```

This should be fixed by requiring edwards v1.0.0 and publishing the fix as chaincfg/v1.5.1.

--- a/chaincfg/go.mod
+++ b/chaincfg/go.mod
@@ -5,7 +5,7 @@ go 1.11
 require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/decred/dcrd/chaincfg/chainhash v1.0.1
-	github.com/decred/dcrd/dcrec/edwards v0.0.0-20190130161649-59ed4247a1d5
+	github.com/decred/dcrd/dcrec/edwards v1.0.0
 	github.com/decred/dcrd/dcrec/secp256k1 v1.0.1
 	github.com/decred/dcrd/wire v1.2.0
 )

